datadog_checks_downloader/requirements.in

```diff
@@ -1,12 +1,15 @@
 #
-# At the time of writing (Jan 30 2020), this was the latest version of these
+# At the time of writing (Sept 24 2021), this was the latest version of these
 # libraries.
 #
-tuf==0.17.0
-in-toto==1.0.1
+tuf==0.17.0; python_version < "3.0"
+tuf==0.18.1; python_version > "3.0"
+in-toto==1.0.1; python_version < "3.0"
+in-toto==1.1.1; python_version > "3.0"
```

Comment:
IIRC, last ceremony we used latest py2-py3 compatible version because spec _may_ be changing between 2 package versions, which could cause package verification issues. @trishankatdatadog to confirm this, but we should do this upgrade more carefully

Reply:
Should this be >= instead of >?

Reply:
No, 3.8 > 3.0